src/app/CASESessionManager.h

```diff
@@ -80,7 +80,7 @@ class CASESessionManager : public OperationalSessionReleaseDelegate, public Sess
      * setup is not successful.
      */
     void FindOrEstablishSession(const ScopedNodeId & peerId, Callback::Callback<OnDeviceConnected> * onConnection,
-                                Callback::Callback<OnDeviceConnectionFailure> * onFailure
+                                Callback::Callback<OnDeviceConnectionFailure> * onFailure, bool useTransportForLargePayload = false
```

Comment:
This is a source break in public API, for consumers that use CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES, no?

Reply:
Moved the new arg to the end with a default value.